README.md

```diff
@@ -129,7 +129,8 @@ Available options (and their defaults)
 {
   "doc": {
     "url_style": "default", // can also be "json"
-    "disable_title_and_description": false // remove the title and the description, useful when using your own custom header
+    "disable_title_and_description": false, // remove the title and the description, useful when using your own custom header
+    "toc": true // insert the table of content for json scheme documentation to the top of the file
```

Comment:
Since this changes behavior, maybe we should default to false for backwards compatibility? (at least initially)